docs/bitwise-repro.rst

```diff
@@ -0,0 +1,16 @@
+.. meta::
+  :description: rocThrust documentation and API reference
+  :keywords: rocThrust, ROCm, API, reference, data type, support
+  
+.. _bitwise-repro:
+
+******************************************
+Bitwise reproducibility
+******************************************
+
+With the exception of the below functions, all rocThrust API functions are bitwise reproducible - that is, given identical inputs, the function will return the exact same result in repeated invocations.

```

Comment:
Please add period after reproducible so that it looks like this:

... reproducible. That is, given...

Reply:
Done.